metricbeat/module/ceph/_meta/config.reference.yml

```diff
@@ -0,0 +1,5 @@
+- module: ceph
+  metricsets: ["cluster_disk", "cluster_health", "monitor_health", "pool_disk", "osd_tree"]
```

Comment:
This is another example of module where we may want different default periods for different metricsets.

Reply:
good point, let me look into this. I also realised one of the metricsets is experimental so not all of them should be here (or I should set it to beta).